var/spack/repos/builtin/packages/py-unyt/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyUnyt(PythonPackage):
+    """A package for handling numpy arrays with units.
+
+    """
+    homepage = "https://yt-project.org"
+    url      = "https://github.com/yt-project/unyt/archive/v2.8.0.tar.gz"
```

Comment:
```suggestion
   pypi = "unyt/unyt-2.8.0.tar.gz"
```
Update checksums too